card/mod/pointer/lib/javascript/script_pointer_config.js.coffee

```diff
@@ -24,12 +24,13 @@ decko.editorInitFunctionMap['.pointer-list-editor'] = ->
 
 $.extend decko,
   initPointerList: (input) ->
-    optionsCard = input.closest('ul').data('options-card')
-    optionsCard ||= input.closest('.pointer-list-editor').data('options-card')
-    input.autocomplete {
-      source: decko.prepUrl decko.rootPath + '/' + optionsCard +
-          '.json?view=junction_name_complete'
-    }
+    decko.initAutoCardPlete input
+
+  initAutoCardPlete: (input) ->
+    optionsCard = input.data 'options-card'
+    return unless !!optionsCard
```

Comment:
hmm.  better: `if !optionsCard`

Reply:
`unless optionsCard` doesn't work in coffeescript?

Reply:
that is my memory, but I woudn't swear it.